docs/mosip-installation-without-public-dns.md

```diff
@@ -0,0 +1,115 @@
+# MOSIP installation without public DNS
+
+## Prerequisites
+* [openssl](https://www.openssl.org/)
+* Chrome browser
+* windows system
```

Comment:
add the use of windows system.
Mention it will be explicitely used for testing reg-client.